api/src/main/java/org/openmrs/validator/UserValidator.java

```diff
@@ -126,6 +126,9 @@ public void validate(Object obj, Errors errors) {
 					errors.rejectValue("username", "error.username.pattern");
 				}
 			}
+			if(!isEmailValid(user.getEmail())) {
```

Comment:
I think email is optional so you should only validate the email when it's provided

Reply:
Ok that's true